README.md

```diff
@@ -11,7 +11,10 @@ This script creates a new Custom role "Custom Role - Cloud One Conformity" withi
 This new Custom role along with the built-in "Reader" role will be applied to either all the Subscriptions in the
 Active Directory or just the subscription that is specified.
 
-The script can also be used to create a new App Registration with the default name "Conformity Azure access".
+The script can also be used to create a new App Registration with the default name "Conformity Azure access".Make sure there has no existing App with the default name.
```

Comment:
🔧 I don't think that this additional information is needed at this point as this is just an overview and isn't about running the script. 

I also think that scenario is already covered in the readme below:

> If you choose to use the "Conformity Azure access" App Registration, and it hasn't been created before, the script with automatically create it and add the required API permissions for Conformity to run the rule checks.

Though it might be a good idea to just keep this generic and update it a bit.

```suggestion
The script can also create a new App Registration to apply these permissions which can be used to allow Conformity to access to your Azure instance.
```